content/spaces/space.textile

```diff
@@ -0,0 +1,235 @@
+| location | The current "location":/spaces/locations of the member. | Object |
+| lastEvent.name | The most recent event emitted by the member. | String |
+| lastEvent.timestamp | The timestamp of the most recently emitted event. | Number |
+
+h3(#unsubscribe). Unsubscribe from space events
+
+Unsubscribe from space events to remove previously registered listeners.
+
+The following is an example of removing a listener:
+
+```[javascript]
+space.unsubscribe('update', listener);
+```
+
+h2(#retrieve). Retrieve space state
+
+The current state of the space can be retrieved in a one-off call. This will return an array of all @member@ objects currently in the space. This is a local call and retrieves the membership of the space retained in memory by the SDK.
+
+The following is an example of retrieving the current space state. Ths includes members that have recently left the space, but have not yet been removed:
+
+```[javascript]
+const spaceState = await space.getState();
+```
+
+h2(#advanced). Advanced
+
+The following sections are only relevant if you want to further customize a space, or understand more about the Spaces SDK. They aren't required to get up and running with the basics.
+
+h3(#options). Space options
+
+An additional set of optional properties may be passed when "creating or retrieving":#create a space to customize the behavior of different features.
+
+The following properties can be customized:
+
+|_. Property |_. Description |_. Type |
+| offlineTimeout | Number of milliseconds after a member loses connection or closes their browser window to wait before they are removed from the member list. The default is 120,000ms (2 minutes). | Number |
+| cursors | A "cursor options":/spaces/cursors#options object for customizing live cursor behavior. | Object |
+| cursors.outboundBatchInterval | The interval, in milliseconds, at which a batch of cursor positions are published. This is multiplied by the number of members in a space, minus 1. The default value is 100ms. | Number |
+| cursors.paginationLimit | The number of pages searched from history for the last published cursor position. The default is 5. | Number |
+
+The following is an example of customizing the space options when calling @spaces.get()@:
+
+```[javascript]
+const space = await spaces.get('slideshowPresentation', {
+	offlineTimeout: 180_000,
+	cursors: { paginationLimit: 10 }
+});
+```
+
+h3(#foundations). Space foundations
+
+The Spaces SDK is built upon existing Ably functionality available in Ably's Core SDKs. Understanding which core features are used to provide the abstractions in the Spaces SDK enables you to manage space state and build additional functionality into your application.
+
+A space is created as an Ably "channel":/channels. Members "attach":/channels#attach to the channel and join its "presence set":/presence-occupancy/presence when they "enter":#enter the space. Avatar stacks, member locations and component locking are all handled on this channel.
```

Comment:
I think it's important to mention here that the cursor is a different channel due to the different frequency and batching requirements of cursors.